remoulade/middleware/retries.py

```diff
@@ -104,7 +115,13 @@ def after_process_message(self, broker, message, *, result=None, exception=None)
             message.fail()
             return
 
-        new_message = message.copy()
+        retry_queue_name = message.queue_name
+        escalation_queues_order = self.get_option("escalation_queues_order", broker=broker, message=message)
+
+        if escalation_queues_order and (escalation_map := escalation_queues_order.get(message.queue_name)):
```

Comment:
as you see below we only want to do it once new_message.options["increase_priority_on_retry"] = False  # we only want to do it once